permissions/UDisksListen.permission

```diff
@@ -0,0 +1,18 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailfish-catalog
+# x-sailjail-translation-key-description = permission-la-udisks2_listen
+# x-sailjail-description = Use UDisks2 for listening
```

Comment:
Similarly here: there shouldn't be UDisks2 here. Perhaps this shouldn't have any translations and instead it should be an internal permission that's just included from RemovableMedia.

Reply:
Better with 94588c1489e297c32c6da6ee193a4119ed181cc6 ?